WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/PointOfSaleLoadingView.swift

```diff
@@ -21,6 +41,9 @@ private extension PointOfSaleLoadingView {
         static let textSpacing: CGFloat = 16
         static let progressViewSpacing: CGFloat = 72
     }
+    enum Constants {
+        static let toMilliseconds: Double = 1000.0
```

Comment:
nit: this doesn't seem needed anymore.

Reply:
Updated! https://github.com/woocommerce/woocommerce-ios/pull/15047/commits/9c57d76ed72a09739a06b811abbcbd9af6ef60b0